formbricks/formbricks
# fix(bug): removed user from Team, but user can select team

## What does this PR do?

fixes removed user don't have team access after removing them from the team.
the team-name will not available in this team list.

**NOTE : May be not the best solution and approach**

Fixes #983 

GD VIDEO : https://drive.google.com/file/d/1h8HnX9PKOo0NWyhuwzs1hAYLTfn5sl5-/view?usp=sharing

## Type of change




## How should this be tested?



- pnpm clean && pnpm build
- pnpm go
- user1 create new team and invite user2
- login as user2 and confirm from mail 
- (as user2) you would be able to see team name in team list created by user1
- (as user2) switch the team to user1 created team
- now logout and login as user1 and remove user2 from that specific group.
- now login again as user2 and check u would not be able to see team created by user1

## Checklist



### Required


### Appreciated
